casper/src/test/resources/RegistryTest.rho

```diff
@@ -42,7 +43,6 @@ match (
           ("InsertArbitrary and lookup should work", *test_insertArbitraryAndLookup),
           ("InsertArbitrary should work with increased nonce", *test_insertSignedValidNonce),
           ("InsertArbitrary should return Nil for invalid nonce", *test_insertSignedInvalidNonce),
```

Comment:
Please correct text description for these two tests. They are testing Insert signed not arbitrary.

---

Please also add test for nonce with `Long.MaxValue + 1` which should still be incorrect so we have the same situation tested as with blessed contracts.